contributing/getting-started-e2e.md

```diff
@@ -216,15 +216,15 @@ In the workflow we will be using you'll go to the master branch on your local re
 git checkout master
 
 # pull in the latest changes from the remote amphtml repository
-git pull upstream master 
+git pull upstream master
```

Comment:
Yes; for the Getting Started guides we lead people through setting up the upstream alias to that.  More advanced users can set it to whatever they want, but for the most part in these guides we try to stay away from providing too many options. :)